code/game/objects/items.dm

```diff
@@ -76,6 +76,9 @@
 
 /obj/item/New()
 	..()
+	initialize_action_buttons()
```

Comment:
Why do you need to initialize some action buttons in a special way ?

Reply:
Because action buttons are fucking retarded and if you want to add a new action button to an atom after its New() you also need to call this.